jigsaw

```diff
@@ -47,13 +51,23 @@ $container->bind(BladeHandler::class, function ($c) {
     return new BladeHandler($c[Factory::class]);
 });
 
+$container->bind(MarkdownParser::class, \Mni\FrontYAML\Bridge\Parsedown\ParsedownParser::class);
+$container->bind(YAMLParser::class, \Mni\FrontYAML\Bridge\Symfony\SymfonyYAMLParser::class);
```

Comment:
For consistency, let's import these classes in the `use` block and just refer to the final class names `ParsedownParser::class` and `SymfonyYAMLParser::class` here.